From fab68a265e6ebc0b048e70b84ca9f416955c3c8a Mon Sep 17 00:00:00 2001
From: Paul Naoumenko <paulx.naoumenko@intel.com>
Date: Fri, 2 Aug 2013 08:59:27 +0200
Subject: [PORT FROM MAIN]wifi: add permissions for wifi to property

BZ: 99166 111626

wpa_supplicant needs wifi permission to set its pid in
a user defined property

Change-Id: I45da34c923313822696d7fa863a4a670a035a5d5
Orig-Change-Id: I726dde0c53a76d9130e8554eb66cdb6b62be174c
Category: device enablement
Domain: CWS.WIFI-Common
Origin: upstream
Upstream-Candidate: no,proprietary
Signed-off-by: Jean-Michel Bachot <jean-michel.bachot@intel.com>
Reviewed-on: http://android.intel.com:8080/122934
Reviewed-by: Poulain, LoicX <loicx.poulain@intel.com>
Reviewed-by: Naoumenko, PaulX <paulx.naoumenko@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 init/property_service.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/init/property_service.c b/init/property_service.c
index e52dd33..c0201a2 100644
--- a/init/property_service.c
+++ b/init/property_service.c
@@ -81,6 +81,7 @@ struct {
     { "sys.powerctl",     AID_SHELL,    0 },
     { "service.",         AID_SYSTEM,   0 },
     { "wlan.",            AID_SYSTEM,   0 },
+    { "wpa_supplicant.",  AID_WIFI,     0 },
     { "bluetooth.",       AID_BLUETOOTH,   0 },
     { "dhcp.",            AID_SYSTEM,   0 },
     { "dhcp.",            AID_DHCP,     0 },
-- 
1.7.4.1

